app/schemas/models/user.js

```diff
@@ -402,6 +402,11 @@ _.extend(UserSchema.properties, {
   seenNewDashboardModal: { type: 'boolean', description: 'Whether the user has seen the new dashboard onboarding modal? Set to true after the modal is seen and closed by the user' }, // Ozaria
   closedNewTDGetStartedTooltip: { type: 'boolean', description: 'Whether the user has closed the get started tooltip in the new dashboard? Set to true once the user has dismissed the tooltip' }, // Ozaria
 
+  dashboardVersion: {
```

Comment:
I dont think keeping it at top-level is a good idea. What issue did you face with features? Could create a new property `flags` if you need. We have properties like this: `seenNewDashboardModal`, `closedNewTDGetStartedTooltip` already which we should club into a single object(later work)